extensions/amp-bind/0.1/test/integration/test-bind-impl.js

```diff
@@ -99,7 +99,7 @@ function waitForEvent(env, name) {
   });
 }
 
-describe.configure().ifNewChrome().run('Bind', function() {
+describe.configure().ifIe().ifNewChrome().run('Bind', function() {
```

Comment:
Yea, we could prob do this with something like `describe.configure().on(['ie', 'new-chrome'].run(...)`.